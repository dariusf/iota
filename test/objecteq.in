Person := Object clone
writeln(Person == Person)