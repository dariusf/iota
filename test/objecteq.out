_io.Lobby.send('Lobby').send('setSlot', (_io.IoStringWrapper('Person')), (_io.Lobby.send('Object').send('clone')));
_io.Lobby.send('writeln', (_io.Lobby.send('Person').send('==', (_io.Lobby.send('Person')))));